packages/@webex/plugin-meetings/src/meetings/util.ts

```diff
@@ -220,4 +223,39 @@ MeetingsUtil.checkH264Support = async function checkH264Support(options: {
   }, delay);
 };
 
+/**
+ * get device from locus data
+ * @param {Object} newLocus new locus data
+ * @returns {Object}
+ */
+MeetingsUtil.getThisDevice = (newLocus: any) => {
+  if (newLocus?.self?.devices?.length > 0) {
+    const [thisDevice] = newLocus.self.devices;
```

Comment:
question: is it possible to have multiple devices? I ask because this method works differently to how it is done in selfUtils. In self utils the self.devices is searched for a device that matches `this.webex.internal.device.url`